presto-native-execution/src/test/resources/external_functions.json

```diff
@@ -36,6 +36,102 @@
           "isOrderSensitive": false
         }
       }
+    ],
+    "avg": [
```

Comment:
We should not overwrite existing Presto functions. Let's remove this entry.

Reply:
Thanks for the review! I thought all functions needs to be registered through this JSON file. Let me double check with @Ali-P.

Reply:
- This UDF is accessible only by `json.test_schema.avg(...)` as of now (so, it doesn't overwrite `presto.default.avg(...)`)
- You need to register the functions you use in the queries. I don't see you using `json.test_schema.avg` UDF in the queries. So, you can remove `avg` entry from the json file UNLESS you've listed it as **Documentation** (which would make sense to keep the registration).